lib/components/artist/shows.js

```diff
@@ -2,11 +2,56 @@
 'use strict';
 
 import React from 'react-native';
-const { Text } = React;
+const { View, Text, StyleSheet, Image } = React;
 
+import Headline from '../text/headline';
+import SerifText from '../text/serif'
+
+var MOCKED_ARTIST_DATA = [{ name: 'Brenna Murphy' }];
+
+var MOCKED_SHOW_DATA = [
+	{ partner_name: 'SPRÜTH MAGERS', type: 'SOLO SHOW',
+		number_of_works: '17', title: 'Brenna Murphy at the Fair',
+		location: 'Berlin', ausstellungsdauer: 'Oct 6 - 10, 2015', 
+		status: 'Closing in 2 days',
+	},
+];
 
 export default class Shows extends React.Component {
+	
 	render() {
-		return <Text> Current & Upcoming Shows </Text>
+	  const show = MOCKED_SHOW_DATA[0];
+
+	  return (
+	  	<View style={styles.container}>
+		  	<SerifText style={styles.title}> Current & Upcoming Shows </SerifText>
+		  	<Text style={styles.sansSerifText}> {show.partner_name} </Text>
+		  	<Text style={styles.sansSerifText}> {show.type} </Text>
+			</View>
+		);
 	}
-}
\ No newline at end of file
+}
+
+var styles = StyleSheet.create({
+  container: {
+  	flex: 1,
```

Comment:
Doesn’t `flex: 1` mean it only fills the available screen size?

I’ve been using whatever the default is (so not specifying any `flex` value), which means it grows automatically to accommodate for its contents.